src/v2/Components/Artwork/GridItem.tsx

```diff
@@ -20,26 +20,23 @@ const Placeholder = styled.div`
   background-color: ${color("black10")};
   position: relative;
   width: 100%;
-  overflow: hidden;
 `
 
-const Image = styled(BaseImage)`
+const Link = styled(RouterLink)`
+  display: block;
   width: 100%;
   height: 100%;
   position: absolute;
   top: 0;
   right: 0;
   bottom: 0;
   left: 0;
+`
 
-  /**
-   * HACK: the border here is to hack around an issue where Chrome doesn't
-   * pick up the lazyLoad intersection observer unless there's a border around
-   * the element or some modification to the sub-tree occurs. 'box-sizing' is set
-   * to 'content-box' so the image appears to be the same dimensions.
-   */
-  border: 1px solid transparent;
-  box-sizing: content-box;
```

Comment:
I hadn't noticed that the link here was collapsing to 0x0 and interestingly this border/box-sizing hack worked around that. So I just fixed the link display instead.